### PR TITLE
Jn peak find algo

### PR DESCRIPTION
**Pull request by jan2nov**
---
name: Pull Request
about: Integration of initial stencil 7x7 algorithm and peak filtering.

---

**Description of pull request**
The new features enable to run a stencil 7x7 code with peak filtering algorithm. This way it selects only local peaks of the candidates

**Interfaces**
Will this pull request result in a backwards-incompatible interface change: no

Expected semantic version number increment category (Please indicate x.y.z): y

**Issues this pull request solves**

**New tag of master**

**New deployment**

**Keep branch after merging**

**Notes**

### CODE REVIEW

File: include/aa_params.hpp [hunk 1]
```diff
@@ -78,6 +78,11 @@ namespace astroaccelerate {
 #define DIT_YSTEP 2
 #define DIT_ELEMENTS_PER_THREAD 4
 
+//experimental clustering filter
+#define PPF_PEAKS_PER_BLOCK 10
+#define PPF_DPB 128
+#define PPF_SIZE_SEARCH 15
```
Comment (KAdamek) [inline include/aa_params.hpp]: I would change PPF_SIZE_SEARCH to PPF_SEARCH_RANGE_IN_MS to indicate that it is in ms maybe add comment as well.

**Review by KAdamek — CHANGES_REQUESTED** [review]
Just one thing regarding name of the #define variable

File: include/aa_params.hpp (hunk 1 above)
Comment (jan2nov) [reply include/aa_params.hpp]: Done

**Comment by jan2nov** [comment]
During compilation there where warnings of unused variables (aa_pipeline_generic.hpp:30:58: warning: unused parameter ‘analysis_enable_candidate_algorithm’) -- Think this variable is not needed now as it moved to the options.

**Review by KAdamek — APPROVED** [review]
Looks OK

**Pull request merged**